evap/results/tests/test_views.py

```diff
@@ -26,7 +26,7 @@ def setUpTestData(cls):
 
 
 class TestResultsSemesterCourseDetailView(ViewTest):
-    url = '/results/semester/2/course/21'
+    url = '/results/semester/1/course/1'
```

Comment:
there is one automatic test that asserts that the view doesn't crash. if uses the data created in setUpTestData. in there, a course with id 21 is created, which is why this worked before.

please don't use the fixtures, i planned to remove them. what was the problem with creating the data via model mommy?